db/models.py

```diff
@@ -50,3 +59,78 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    username = models.CharField(max_length=68, unique=True)
+    password = models.CharField(max_length=68)
+    email = models.EmailField()
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        null=True
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
```

Comment:
```suggestion
        return str(self.created_at)
```
In this case better to use str

Reply:
![image](https://github.com/mate-academy/py-tickets-and-orders/assets/52757595/fc65931e-7d31-46cf-bf54-8193d4076557)

Reply:
коли змінив з'явилась така помилка

Reply:
а ні, то моя вина, забув забрати фігурні лапки